website/www/site/assets/js/quotes-slider.js

```diff
@@ -21,7 +21,7 @@ new Swiper('.swiper', {
     bulletActiveClass: "bullet-active-class-custom",
   },
   breakpoints: {
-    1024: {
+    1025: {
```

Comment:
if having 1024 as a breakpoint doesn't work here properly, we can add a comment like:
// switches from mobile to desktop when window width is > 1024px

Reply:
done